src/tool/subcommands/api_cmd.rs

```diff
@@ -729,6 +778,127 @@ async fn compare_apis(
     run_tests(tests, &forest, &lotus, &config).await
 }
 
+async fn start_offline_server(
+    snapshot_path_opt: Option<PathBuf>,
+    chain: NetworkChain,
+    rpc_port: u16,
+    rpc_data_dir: PathBuf,
+) -> anyhow::Result<()> {
+    info!("Configuring Offline RPC Server");
+    let client = Client::default();
+    let db_path = client.data_dir.as_path().join(rpc_data_dir);
+    let db = Arc::new(ParityDb::open(&db_path, &ParityDbConfig::default())?);
+
+    let (snapshot_file, snapshot_path) = if let Some(path) = snapshot_path_opt {
+        (File::open(&path).await?, path)
+    } else {
+        warn!(
```

Comment:
Shall we add some prompt before downloading?

Reply:
yes i'll add a similar prompt

Reply:
Thanks, then I guess we also need a similar `--force` or `--yes` option to bypass the prompt

Reply:
done